plugins/defaultplugins/ifplugin/vppcalls/tap_vppcalls.go

```diff
@@ -71,17 +69,17 @@ func AddTapInterface(ifName string, tapIf *interfaces.Interfaces_Interface_Tap,
 		reply := &tap.TapConnectReply{}
 		err = vppChan.SendRequest(req).ReceiveReply(reply)
 		retval = reply.Retval
-		swIfIndex = reply.SwIfIndex
+		swIfIdx = reply.SwIfIndex
 		msgName = reply.GetMessageName()
 	}
 	if err != nil {
-		return 0, err
+		return swIfIdx, err
 	}
 	if retval != 0 {
-		return 0, fmt.Errorf("%s returned %d", msgName, retval)
+		return swIfIdx, fmt.Errorf("%s returned %d", msgName, retval)
```

Comment:
Same applies here, always better to read zero-ed value if non nil error is returned.